src/analysis/convolution.lean

```diff
@@ -670,10 +670,50 @@ begin
     exact (mul_le_mul_of_nonneg_right op_norm_lsmul_le hε).trans_eq (one_mul ε) }
 end
 
+/-- `(φ i ⋆ g i) (k i)` tends to `z₀` as `i` tends to some filter `l` if
+* `φ` is a sequence of nonnegative functions with integral `1`
+  whose support tends to small neighborhoods around `(0 : G)` as `i` tends to `l`.
+* `g i x` tends to `z₀` as `(i, x)` tends to `l ×ᶠ 𝓝 x₀`
+* `k i` tends to `x₀`
+
+This requires that `g` is locally integrable, which is a bit stronger than the condition in
+`convolution_tendsto_right`. -/
+lemma convolution_tendsto_right' [sigma_compact_space G]
+  {ι} {g : ι → G → E'} {l : filter ι} {x₀ : G} {z₀ : E'}
+  {φ : ι → G → ℝ} {k : ι → G}
+  (hnφ : ∀ i x, 0 ≤ φ i x)
+  (hiφ : ∀ i, ∫ x, φ i x ∂μ = 1)
+  (hφ : tendsto (λ n, support (φ n)) l (𝓝 0).small_sets)
+  (hig : ∀ j, locally_integrable (g j) μ)
```

Comment:
I don't understand why you would need local integrability instead of, say, ae measurability: the assumptions `hcg` implies that `g` converges to something, so it is asymptotically locally bounded, right?

Reply:
Oh you're right. In fact, in the current proof I only use `hig` to show that `g j` is ae-strongly measurable. This is a left-over from a previous proof strategy where I wanted to use the continuity of the convolution at `x₀`. Thanks for spotting this!